tests/backup/backup_helper.go

```diff
@@ -1254,6 +1292,213 @@ func IsBackupLocationPresent(bkpLocation string, ctx context.Context, orgID stri
 	return false, nil
 }
 
+// GetCurrentScheduleBackupCount returns the current number of schedule backups for the given schedule
+func GetCurrentScheduleBackupCount(ctx context.Context, scheduleName string, orgID string) (int, error) {
+	allScheduleBackupNames, err := Inst().Backup.GetAllScheduleBackupNames(ctx, scheduleName, orgID)
+	if err != nil {
+		return -1, err
+	}
+	return len(allScheduleBackupNames), nil
+}
```

Comment:
There is no need for this function. If some test needs this, it is easier to call this code

```
allScheduleBackupNames, err := Inst().Backup.GetAllScheduleBackupNames(ctx, scheduleName, orgID)
if err != nil {
	return -1, err
}
```

Reply:
Very well. This helps a bit in reducing code duplication and avoid unnecessary storage of all backup names in memory.

Reply:
As discussed, we can remove this

Reply:
Removed GetCurrentScheduleBackupCount.